tests/integration_tests/cpu/propagator/covariance_transport.cpp

```diff
@@ -354,7 +357,7 @@ TYPED_TEST(detray_propagation_HelixCovarianceTransportValidation,
     for (unsigned int i = 0u; i < e_bound_size; i++) {
         EXPECT_NEAR(bound_vec_0[i], bound_params[i], this->tolerance)
             << "i: " << i << "\n"
-            << bound_params;
+            << std::setprecision(8) << bound_params;
```

Comment:
Unrelated change?

Reply:
It was useful during debugging, I thought I would leave it in. But let me open a separate PR after we have the new algebra-plugins version (which also contains better printing)